src/components/PageHeader/PageHeader.module.css

```diff
@@ -19,6 +19,14 @@
   }
 }
 
+.page-header--50-50 {
+  @media all and (min-width: $eds-bp-md) {
+    > div {
```

Comment:
Let's try to use `left` and `right` for this instead of nested tags

```
.page-header__left {
  .page-header--2up & {
    width: 50%;
  }
}

.page-header__right {
  .page-header--2up & {
    width: 50%;
  }
}
```

CSS gets kind of messy if you nest a bunch of HTML tags. Get a little lost (at least I do)

Reply:
Used specific classes over general elements